Userland/Libraries/LibJS/Runtime/Intl/NumberFormat.cpp

```diff
@@ -377,6 +377,13 @@ static ALWAYS_INLINE int log10floor(Value number)
     return as_string.length() - 1;
 }
 
+static Value subtract(GlobalObject& global_object, Value lhs, Value rhs)
+{
+    if (lhs.is_number())
+        return Value(lhs.as_double() - rhs.as_double());
+    return js_bigint(global_object.vm(), lhs.as_bigint().big_integer().minus(rhs.as_bigint().big_integer()));
+}
```

Comment:
Hmm, can we not use `sub()` from `Value.h` (with `MUST`) for this?

Reply:
Probably can, but all of these helpers are going away soon

:thousandyakstare: https://tc39.es/proposal-intl-numberformat-v3/out/numberformat/proposed.html#sec-tointlmathematicalvalue